yaptide/routes.py

```diff
@@ -110,65 +105,112 @@ def get():
-# (this is an example route, demonstration pourpose only)
-example_user_args = reqparse.RequestParser()
-example_user_args.add_argument(
-    "name", type=str, help="Example user name is required and must be a string.", required=True)
-
-example_user_modify_args = reqparse.RequestParser()
-example_user_modify_args.add_argument(
-    "name", type=str, help="Example user name is required and must be a string.")
+class UserRegister(Resource):
+    """Class responsible for user registration"""
 
-example_user_fields = {
-    'id': fields.Integer,
-    'name': fields.String,
-}
+    class _Schema(Schema):
+        """Class specifies API parameters"""
 
+        login_name = fld.String()
+        password = fld.String()
 
-class ExampleUserResource(Resource):
-    @marshal_with(example_user_fields)
-    def get(self, user_id):
-        user = ExampleUserModel.query.get_or_404(
-            user_id, description=f"User with id {user_id} not found.")
-        return user
+    @staticmethod
+    def put():
+        """Method returning status of registration"""
+        try:
+            json_data: dict = UserRegister._Schema().load(request.get_json(force=True))
+        except ValidationError:
+            return {
+                'status': 'ERROR',
+                'message': 'Wrong data types provided'
+            }, api_status.HTTP_400_BAD_REQUEST
+
+        try:
+            user = db.session.query(UserModel).filter_by(login_name=json_data.get('login_name')).first()
+        except Exception:  # skipcq: PYL-W0703
+            return {
+                'status': 'ERROR',
+                'message': 'Internal server error'
+            }, api_status.HTTP_500_INTERNAL_SERVER_ERROR
 
-    @marshal_with(example_user_fields)
-    def put(self):
-        args = example_user_args.parse_args()
-        user = ExampleUserModel(name=args.name)
-        db.session.add(user)
-        db.session.commit()
         if not user:
-            abort(500, "Something went wrong.")
-        return user, 201
-
-    @marshal_with(example_user_fields)
-    def delete(self, user_id):
-        user = ExampleUserModel.query.get_or_404(
-            user_id, description=f"User with id {user_id} not found.")
-        db.session.delete(user)
-        db.session.commit()
-        return user, 202
-
-    @marshal_with(example_user_fields)
-    def patch(self, user_id):
-        args = example_user_modify_args.parse_args()
-        user = ExampleUserModel.query.get_or_404(
-            user_id, description=f"User with id {user_id} not found.")
-        for key, value in args.items():
-            if value:
-                setattr(user, key, value)
-        db.session.commit()
-        print(user)
-        return user, 202
-
-############################################
+            try:
+                user = UserModel(
+                    login_name=json_data.get('login_name')
+                )
+                user.set_password(json_data.get('password'))
```

Comment:
store hashed version of the password whit something like bcrypt or other modern hashing algorithms

Reply:
It is, check `set_password` method

Reply:
Okey :D I missed it.